datamegh/api/http.py

```diff
@@ -1 +1,38 @@
-""" HTTP Connection API. """
+""" Basic HTTP verbs for your API connection. """
+
+import requests
+
+
+def get(url, params=None, **kwargs):
+    response = requests.get(url, params, **kwargs)
+    return response
```

Comment:
You can directly return as there aren't any intermediate steps here.  
```suggestion
    return requests.get(url, params, **kwargs)
```